packages/vaadin-custom-field/test/helper.test.js

```diff
@@ -102,4 +102,26 @@ describe('helper text', () => {
       expect(customField.getAttribute('has-helper')).to.be.equal('slotted');
     });
   });
+
+  describe('slotted component', () => {
+    let field, helper;
+
+    beforeEach(() => {
+      field = fixtureSync(`
+        <vaadin-custom-field>
+            <input type="text" />
+
+            <input type="text" slot="helper" />
+        </vaadin-custom-field>
+      `);
+      helper = field.querySelector('[slot="helper"]');
+      field.focus();
+    });
+
+    it('should get focus when clicked', () => {
+      helper.focus();
```

Comment:
```suggestion
      helper.click();
```

Reply:
@hadivaadin please check this test, it needs to be updated as suggested.

Reply:
Actually it might be better to use a slightly different approach as `click()` might not change `activeElement`.

```js
it('should not focus the field on helper click', () => {
  const spy = sinon.spy(field, 'focus');
  helper.click();
  expect(spy.called).to.be.false;
});
```